### ISSUE TITLE
pip install doesn't appear to be working anymore for system wide package installs

### ISSUE THREAD

**Issue by Asriel115** [Ahmed-Al-Balochi/LibreGaming#31]
This appears to be new.... 
![Screenshot from 2023-08-13 22-53-32](https://github.com/ahmed-al-balochi/LibreGaming/assets/63687010/0035a3f8-c359-42e6-b721-6e149e16a159)

**Comment by ahmed-al-balochi** [comment]
Thank you for pointing it out, I really appreciate it.

I will be looking for a solution and will create a new release once I am done.

**Comment by Asriel115** [comment]
Awesome! And thank you for this amazing contribution to the open source community!

**Comment by ahmed-al-balochi** [comment]
Thank you for the kind words.

**Comment by ahmed-al-balochi** [comment]
So I finally got time to look into the issue, the solution is to use pipx to install LibreGaming.
I have updated the README file to include the information.
Basically, it's as follows:
1. Install [pipx](https://github.com/ahmed-al-balochi/LibreGaming#dependencies).
2. run ```sudo pipx install LibreGaming```.
3. Create an alias so that you don't write the full path:
```
alias libregaming="/root/.local/pipx/venvs/libregaming/bin/libregaming" 
```

I have already tested it. Please let me know if you have any issues.

**Comment by Asriel115** [comment]
![image](https://github.com/ahmed-al-balochi/LibreGaming/assets/63687010/d513f94a-4ee3-49bd-a914-930aaa6aeacd)
![image](https://github.com/ahmed-al-balochi/LibreGaming/assets/63687010/185e59ee-0332-49d8-a291-ca2dc320a728)

either I'm doing something wrong or its still not working. I am also not super familiar with pipx I rarely used pip as it was. I ran this in a VM because on my host system I just manually installed and setup everything I needed already. I apologize for my amature approach I am new to a lot of this stuff still.
